README.md

```diff
@@ -185,6 +185,34 @@ const config: Config.InitialOptions = {
 export default config;
 ```
 
+### AnnotationReporter
+
+> ❗️ Do **NOT** configure multiple reporters as this will break your project in unexpected ways!
+
+#### Enable and Configure the AnnotationReporter in `jest.config.ts`
+Add the reporter to the `UiControlClient` in `jest.setup.ts`:
+
+```typescript
+import { AskUIAnnotationStepReporter, AnnotationLevel } from '@askui/askui-reporters';
+...
+  aui = await UiControlClient.build({
+    ...
+    reporter: new AskUIAnnotationStepReporter(
+      {
+        annotationLevel: AnnotationLevel.ON_FAILURE,
+        // folderPath: "report", /* Uncomment and change property for different folder */
+        // fileNameTemplate: "_testStep_annotation" /* Uncomment and change property for different fileNameTemplate */
+      }),
+  });
+...
+```
+
+`AnnotationLevel` is implemented as an enum. You have three options: `DISABLED, `ON_FAILURE`, `ALL`.
```

Comment:
```suggestion
`AnnotationLevel` is implemented as an enum. You have three options: `DISABLED`, `ON_FAILURE`, `ALL`.
```

I think we should remove the `DISABLED` value (from enum) as this is the same as not passing the reporter at all... Why would you pass it in the first place if you disable it afterwards? I mean this is where our API is a little different from before but I think it is okay in this case as people who used it, did not use `DISABLED`...